src/quick-tour/build-loyalty.md

```diff
@@ -21,3 +24,5 @@ Punch Out Solutions
 [1]: https://marketplace.magento.com/
 [2]: https://www.punchoutcatalogs.com/
 [3]: https://www.punchout2go.com/
+[4]: ../customers/account-dashboard-my-purchase-orders.html
```

Comment:
These do work, technically, but let's stick with our standard of {% link file/path.md %} for internal links.

Reply:
Fixed